gulpfile.babel.js

```diff
@@ -91,129 +84,78 @@ const SOURCES = [
+   .pipe(gulp.dest('dist'));
+}
 
-// Compile and Automatically Prefix Stylesheet Templates (production)
-gulp.task('styletemplates', () => {
-  // For best performance, don't add Sass partials to `gulp.src`
-  return gulp.src('src/template.scss')
-    // Generate Source Maps
-    .pipe($.sourcemaps.init())
-    .pipe($.sass({
-      precision: 10,
-      onError: console.error.bind(console, 'Sass error:')
-    }))
-    .pipe($.cssInlineImages({webRoot: 'src'}))
-    .pipe($.autoprefixer(AUTOPREFIXER_BROWSERS))
-    .pipe(gulp.dest('.tmp'))
-    // Concatenate Styles
-    .pipe($.concat('material.css.template'))
-    .pipe(gulp.dest('dist'))
-    // Minify Styles
-    .pipe($.if('*.css.template', $.csso()))
-    .pipe($.concat('material.min.css.template'))
-    .pipe($.header(banner, {pkg}))
-    .pipe($.sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'))
-    .pipe($.size({title: 'styles'}));
-});
+// Compile and Automatically Prefix Stylesheets (production)
+function mdlCss() {
+  const stream = gulp.src('src/material-design-lite.scss')
+    .pipe($.rename('material.css'));
+
+  return cssPipeline(stream);
+}
 
 // Compile and Automatically Prefix Stylesheets (production)
-gulp.task('styles', () => {
-  // For best performance, don't add Sass partials to `gulp.src`
-  return gulp.src('src/material-design-lite.scss')
-    // Generate Source Maps
-    .pipe($.sourcemaps.init())
-    .pipe($.sass({
-      precision: 10,
-      onError: console.error.bind(console, 'Sass error:')
-    }))
-    .pipe($.cssInlineImages({webRoot: 'src'}))
-    .pipe($.autoprefixer(AUTOPREFIXER_BROWSERS))
-    .pipe(gulp.dest('.tmp'))
-    // Concatenate Styles
-    .pipe($.concat('material.css'))
-    .pipe($.header(banner, {pkg}))
-    .pipe(gulp.dest('dist'))
-    // Minify Styles
-    .pipe($.if('*.css', $.csso()))
-    .pipe($.concat('material.min.css'))
-    .pipe($.header(banner, {pkg}))
-    .pipe($.sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'))
-    .pipe($.size({title: 'styles'}));
-});
+function mdlThemeTemplate() {
+  const stream = gulp.src('src/template.scss')
+    .pipe($.rename('material.template.css'));
+
+  return cssPipeline(stream);
+}
 
 // Only generate CSS styles for the MDL grid
-gulp.task('styles-grid', () => {
-  return gulp.src('src/material-design-lite-grid.scss')
-    .pipe($.sass({
-      precision: 10,
-      onError: console.error.bind(console, 'Sass error:')
-    }))
-    .pipe($.autoprefixer(AUTOPREFIXER_BROWSERS))
-    .pipe(gulp.dest('.tmp'))
-    // Concatenate Styles
-    .pipe($.concat('material-grid.css'))
-    .pipe($.header(banner, {pkg}))
-    .pipe(gulp.dest('dist'))
-    // Minify Styles
-    .pipe($.if('*.css', $.csso()))
-    .pipe($.concat('material-grid.min.css'))
-    .pipe($.header(banner, {pkg}))
-    .pipe(gulp.dest('dist'))
-    .pipe($.size({title: 'styles-grid'}));
-});
+function mdlGrid() {
+  const stream = gulp.src('src/material-design-lite-grid.scss')
+    .pipe($.rename('material-grid.css'));
```

Comment:
Hm you are totally right. This is a left-over of porting the old gulpfile where grid had its own task. Will fix.